definitions/variable/technology/power-plant.yaml

```diff
@@ -257,5 +238,4 @@
 - Stored Energy Inventory|Electricity|Energy Storage System:
```

Comment:
SHouldn't this be in electricity-operation?

Reply:
Got it! I'll move it.